com.codemouse92.timecard.yaml

```diff
@@ -0,0 +1,65 @@
+app-id: com.codemouse92.timecard
+default-branch: stable
+base: io.qt.qtwebkit.BaseApp
+base-version: "5.14"
+runtime: org.kde.Platform
+runtime-version: "5.14"
+sdk: org.kde.Sdk
+command: Timecard-App
+cleanup:
+    - /include
+    - /share/aclocal
+    - /share/doc
+    - /share/cmake
+    - /share/man
+    - "*.a"
+    - "*.la"
+finish-args:
+    - --filesystem=host
+    - --share=ipc
+    - --socket=x11
+    - --socket=fallback-x11
+    - --socket=wayland
+    - --device=dri
+    - --talk-name=org.freedesktop.DBus.Proprieties
+    - --talk-name=org.freedesktop.IBus
+    - --talk-name=org.freedesktop.Notifications
+    - --talk-name=org.kde.StatusNotifierWatcher
+    - --own-name=org.kde.*
+    - --talk-name=com.canonical.indicator.application
+    - --talk-name=org.ayatana.indicator.application
+    - --env=PATH=/app/bin
+    - --env=PYTHONPATH=/app/packages/python
+modules:
+    - name: pyside2
+      buildsystem: cmake-ninja
+      builddir: true
+      config-opts:
+          - -DCMAKE_BUILD_TYPE=Release
+          - -DBUILD_TESTS=OFF
+      cleanup:
+          - /bin
+      sources:
+        - type: archive
+          url: https://download.qt.io/official_releases/QtForPython/pyside2/PySide2-5.15.0-src/pyside-setup-opensource-src-5.15.0.tar.xz
+          sha256: f1cdee53de3b76e22c1117a014a91ed95ac16e4760776f4f12dc38cd5a7b6b68
+        - type: shell
+          commands:
+           - mkdir -p /app/include/qt5tmp && cp -R /usr/include/Qt* /app/include/qt5tmp  # https://bugreports.qt.io/browse/PYSIDE-787
+           - sed -i 's|\(--include-paths=\)|\1/app/include/qt5tmp:|' sources/pyside2/cmake/Macros/PySideModules.cmake
+
+    - name: Timecard-App
+      buildsystem: simple
+      build-options:
+        build-args:
+          - "--share=network"
```

Comment:
this will not work. Build bots don't have access to the network. If you need to install python dependencies they need to be added to the manifest.

And the build fails because of this section of the manifest.

Reply:
I'm trying to rewrite where I download and build PySide2 locally. Unfortunately, while it appears to work, I get this on trying to run the program...

> object file has no loadable segments

I'm tearing my hair out. Would you be able to help me diagnose this if I shared the (mostly) revised .yaml?

Reply:
Nevermind! I found your own package, `com.github.unrud.djpdf.yaml`, and it worked. Thanks for leading the way with that, @hfiguriere. :D

Reply:
that's not my package ;-) (just a fork)

but my suggestion often is to search the flathub org on github for other packages that may have solved it.

Reply:
Yeah, I've kinda taken to that approach too.

I really need to write an article about this.